modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -634,6 +634,31 @@ impl WalletClientModule {
         Ok(self.module_api.module_consensus_version().await? >= ModuleConsensusVersion::new(2, 2))
     }
 
+    /// Allocates a deposit address controlled by the federation, guaranteeing
+    /// safe handling of deposits.
+    ///
+    /// This method ensures that the federation can process deposits to the
+    /// allocated address without issues, even for on-chain transactions
+    /// exceeding `ALEPH_BFT_UNIT_BYTE_LIMIT`.
```

Comment:
Should mention it might fail if offline

Reply:
Let me know if the latest version of the docs isn't clear, happy to keep iterating.

Reply:
Sounds good!